### PR TITLE
Add defines of JOB_OBJECT_XXX

### PR DESCRIPTION
**Pull request by mattn**
I hope to build libuv on mingw32.

### CODE REVIEW

File: src/win/process.c [hunk 1]
```diff
@@ -44,6 +45,14 @@
...
 #define E_V(str) { str "=", L##str, sizeof(str), 0, 0 }
 
+#ifndef JOB_OBJECT_LIMIT_BREAKAWAY_OK 
+# define JOB_OBJECT_LIMIT_PROCESS_MEMORY             0x00000100
+# define JOB_OBJECT_LIMIT_JOB_MEMORY                 0x00000200
+# define JOB_OBJECT_LIMIT_DIE_ON_UNHANDLED_EXCEPTION 0x00000400
+# define JOB_OBJECT_LIMIT_BREAKAWAY_OK               0x00000800
+# define JOB_OBJECT_LIMIT_SILENT_BREAKAWAY_OK        0x00001000
+# define JOB_OBJECT_LIMIT_KILL_ON_JOB_CLOSE          0x00002000
+#endif
```
Comment (bnoordhuis) [inline src/win/process.c]: Please wrap each #define in an #ifndef guard. Otherwise LGTM, I think. @piscisaureus?

**Comment by mattn** [comment]
Force pushed.

**Comment by bnoordhuis** [comment]
Looking at it again, I see you added the defines to src/win/process.c. I assume they should go into a header file in src/win.

@piscisaureus Chime in, please.

**Comment by piscisaureus** [comment]
@mattn Yes I think the defines were best insterted here: https://github.com/joyent/libuv/blob/4bdb7d837ad99e59974274bb87fc10a35cbc1bf1/src/win/winapi.h#L4075

@bnoordhuis

**Comment by mattn** [comment]
done

**Comment by bnoordhuis** [comment]
Thanks, landed with a slightly expanded commit log in a5a9b41 and a294e3c.

**Comment by bnoordhuis** [comment]
Sorry, make that bfa3e28 and d7a1ba8.

**Comment by mattn** [comment]
Don't mind :)

**Pull request closed without merging**